lib/executor/externally_controlled.go

```diff
@@ -180,6 +180,11 @@ func (mec ExternallyControlledConfig) NewExecutor(es *lib.ExecutionState, logger
 	}, nil
 }
 
+// HasWork reports whether there is any works for give execution segment.
+func (mec ExternallyControlledConfig) HasWork(es *lib.ExecutionSegment) bool {
+	return es.Scale(mec.MaxVUs.Int64) > 0
```

Comment:
The `externally-controlled` executor should always return `true`, given that you can always initialize new VUs via the REST API

Reply:
Fixed